packages/dx-react-grid-bootstrap4/src/main.css

```diff
@@ -0,0 +1,14 @@
+.dx-rg-bs4-cursor-pointer {
+  cursor: pointer;
+}
+.dx-rg-bs4-user-select-none {
+  user-select: none;
+  -moz-user-select: none;
+  -webkit-user-select: 'none';
+}
+.dx-rg-bs4-opacity-03 {
```

Comment:
I'd prefer to name it `.dx-rg-bs4-inactive` or something like this. I mean, use more abstract name.